File: BackEnd/Service/Models/BlackListModel.cs
namespace Service.Models
{
    public class BlacklistModel
    {
        public Guid BlackListId { get; set; }
        public Guid CandidateId { get; set; }
        public string? Reason { get; set; }
        public DateTime DateTime { get; set; } = DateTime.Now;
        public int Status { get; set; }
        public bool IsDeleted { get; set; } = false;
    }
}

File: FrontEnd/src/app/data/blacklist/blacklist.model.ts
export class BlackList {
	blackListId?: string;
	candidateId?: string;
	reason?: string;
	dateTime?: Date;
	status?: number;
	isDeleted?: boolean;
}
export class BlackListAddModel {
	candidateId?: string;
	reason?: string;
	dateTime?: Date;
	isDeleted?: boolean;
}
